contracts/Removal.sol

```diff
@@ -57,22 +103,29 @@ contract Removal is
     bytes memory data
   ) public override {
     // todo require vintage is within valid year range and doesn't already exist
+    (address market, bytes32 parcelId, bool listNow) = abi.decode(
+      data,
+      (address, bytes32, bool)
```

Comment:
Can we define a struct to describe this 3-tuple?  Similar to this:  https://github.com/nori-dot-eco/contracts/blob/master/contracts/LockedNORI.sol#L251